Polish Validator Community doc (#4117)
w3f/polkadot-wiki: docs/maintain/maintain-guides-validator-community.md

nominators can find nodes when browsing through various dashboards and UIs. When someone interacts
with the chain, it ensures that an address they may come across belongs to the validator, and
actions of that identity throughout various parts of the ecosystem (staking, governance, block
explorers, etc.) form a cohesive representation of their participation.

:::note When running multiple validator nodes, the best way to scale an identity is to use multiple
sub-identities from a single verified identity

:::

It's recommended to fill out as many fields in the identity as possible so Nominators have ample
means of reaching out. Nominators may wish to know more about the Validator, the particular setup,
future staking plans, tooling used, or several additional topics. Having a dedicated website
additionally to provide this sort of information is ideal.

:::note Ledger app doesn't support the extrinsic for setting identity yet

As a workaround, create a primary identity with an on-chain account and then use that primary
identity, assign a [sub-identity](../learn/learn-identity.md#sub-accounts) to the Ledger stash.

:::

### Website

> One thing that can help get some visibility is set up a dedicated site for your validator, which

Reviewer: ```suggestion
One strategy for helping gain additional visibility is to set up a dedicated site for your validator, which
```

includes the networks that one is a validator for and validator details such as addresses,
commission, and so forth. Including all suggestions from this page is potential content to include

> on the site. After setting up a website, a validator should consist of this website in the

Reviewer: ```suggestion
on the site. After setting up a website, a validator should add this website to the
```

corresponding field in their identity so nominators can find it easily.

## Transparency & Establishing Trust

Considering the risks involved for both Validators and Nominators, establishing trust is one of the
most essential factors in running quality validator services.

### Self Stake

Validators should have skin in the game in their operations in the form of a stake that is self-bonded

> to their validator stash. As slashing applies to the total stake of a Validator, having a high self-stake shows confidence in operations, and they have skin in the game and a lot to lose if they

Reviewer: ```suggestion
to their validator stash. Slashing applies to the total stake of a Validator, therefore having a high self-stake shows confidence in the operations. This helps show commitment from the Validator as they have skin in the game and can be penalized for negative actions or poor maintenance.
```

#### Servers

Outlining how a validator runs its servers helps nominators understand how diversified a
validator is. Does the Validator run in the cloud, on dedicated machines, in a co-located
datacenter, or in a home residential setup? Do they run multiple nodes on the same machine? If every
validator is hosted in AWS, there is a risk of potential outages that cause large amounts of nodes
to go offline, causing slashing for unresponsiveness. Nominators may want to choose validators that
have thoroughly diversified the providers they use or the facilities they operate in.

Additionally, how does a Validator contribute to decentralization? It can be helpful to outline

> efforts towards this so that the values of a Nominator and Validator are aligned.

Reviewer: ```suggestion
these efforts so that the values of a Nominator and Validator are aligned.
```

It's also helpful to outline what kind of OS is used on these servers and what is the updating

> policy on the software on that OS. For example, are LTS versions used? Do they use NiXOS,

Reviewer: ```suggestion
policy for the software on that OS. For example, are LTS versions used? Do they use NiXOS,
```

distro-packaged libraries? Any server hardening practices, etc.

#### Specs

Are you running the recommended Standard Hardware for
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}? Can you ensure that machines have
enough processing power, memory, file storage, and network connectivity? It's helpful for nominators
to know the specs of the machines a validator uses to assess how they may perform in the network.
If a validator is running underpowered machines, they may not want to nominate them, as these can
result in fewer blocks produced and fewer overall rewards. In certain circumstances, more powerful
machines can result in higher rewards for both the Validator and their Nominators.

> #### Automation and orchestration approaches (terraform, ansible, chef, puppet, Kubernetes, etc.)

Reviewer: ```suggestion
#### Automation and orchestration approaches (Terraform, Ansible, Chef, Puppet, Kubernetes, etc.)
```

What kind of approach is taken for spinning up and provisioning nodes? How might you automate
spinning up large clusters of nodes and upgrading them? Elaborating on what type of automation (or
lack thereof) can help get a sense of how robust a validator setup is. Many everyday actions
or routine maintenance needs to be done, and automating this type of thing often helps mitigate
human errors.

#### Network Topology

Does the Validator node have protection against Denial of Service attacks, and if so, how is that

> done? Outlining the kind of network topology of Validator infrastructure will help Nominators understand how resilient operations are to attacks. Some things to highlight are the usage of firewalls,

Reviewer: ```suggestion
done? Outlining a desired network topology for a Validators infrastructure design will help Nominators understand how resilient their operations are to attacks. Some things to highlight are the usage of firewalls,
```

VPNs, network segmentation, proxies, or other layers separation.

#### Upgrading

Both Polkadot and Kusama releases are published.
[here](https://github.com/paritytech/polkadot/releases). Validators are expected to upgrade their
nodes as soon as a new release comes. Although not every release is mandatory to upgrade,
each new release usually has bug fixes, optimizations, new features, or other beneficial changes.
It's in the best interest of the entire network that validators update their nodes in a timely
fashion. This signals to nominators that a validator is timely care about their operations and is
quick to adapt to necessary circumstances.

It can also be helpful for Nominators to know how the Validator runs software and where they get new

> binaries. How do they get alerted for new releases? Do they hear about things from the matrix chatrooms? Do

Reviewer: ```suggestion
binaries. How do they get alerted for new releases? Do they receive updates from the matrix chatrooms? Do
```

A diverse network of nodes in varying different regions helps strengthen decentralized networks.
Outlining what regions nodes are in gives clarity to this facet of networks. Nominators may want to
promote validators that actively try to decentralize networks operating in regions in which others
do not run nodes.

##### Security / Key handling policies

It is paramount that session keys and stash/controller keys are stored and handled with the utmost

> As a compromise of these, care can get both validator and nominator slashed. Outlining how keys are

Reviewer: ```suggestion
care.  If compromised, both the validator and nominator can be slashed. Outlining how keys are
````
